### ISSUE TITLE
Repeated crashing

### ISSUE THREAD

**Issue by smalltimer** [gdkrmr/lsp-julia#57]
Hi,
The LSP keeps crashing automatically. The message I get is `LSP :: julia-ls has exited (exited abnormally with code 1)`. I think I am missing something very key in my config. As of now, it simply reads:
```
(use-package lsp-julia
  :config
  (setq lsp-julia-default-environment "~/.julia/environments/v1.6")
  (add-hook 'julia-mode-hook #'lsp-deferred))
```
Any comments or tips? Thanks!!

**Comment by gdkrmr** [comment]
is it #56?

**Comment by smalltimer** [comment]
Hi, I'm not sure. I get this particular error: 
```
Error in post-command-hook (lsp-signature): (error "The connected server(s) does not support method textDocument/signatureHelp.
To find out what capabilities support your server use ‘M-x lsp-describe-session’
and expand the capabilities section")

```
The server is starting, and then turning off on its own whenever it has to display suggestions. I think it may be related to SymbolServer.jl which should be run asynchronously (Julia's language server is abysmal). Maybe it is being run synchronously by lsp-julia. I don't know enough elisp to really dig into this :(

**Comment by gdkrmr** [comment]
In any case we have to wait until there is a new version of LanguageServer.jl tagged, until then I suggest you try out the temporary fix in #56.

**Comment by gdkrmr** [comment]
should be fixed with new release. Requires Julia 1.7
